README.md

```diff
@@ -83,13 +83,29 @@ It is recommended to end the path component of the URL with `/graphql`, for exam
 
 # Serialization Format
 
-Similar to the GraphQL specification this specification does not require a specific serialization format.
-For consistency and ease of notation, examples of the response are given in JSON throughout the spec.
+The GraphQL specification allows for many [serialization formats to be implemented](https://spec.graphql.org/June2018/#sec-Serialization-Format). In practice JSON is the most common serialization format. In order to be compliant, servers and clients implementing GraphQL over HTTP must support JSON serialization.
+
+## Content Types
+
+The following are the officially recognized GraphQL content types to designate encoding JSON over HTTP.
+
+| Name | Description |
+| `application/graphql+json` | Required
+| `application/json` | To support legacy clients
+
+A server MUST support requests from clients with HTTP header `Content-Type: application/graphql+json` indicating that the body of the request is a JSON document with a GraphQL request. A server must indicate the content type of the response with a `Content-Type` header. This default value should be `Content-Type: application/graphql+json` indicating that the response is a valid JSON document containing at least one of the `data` or `errors` as a top-level attribute.
+
+A server MAY support requests from clients with other content types.
+
+A client MUST handle receiving responses with HTTP header `Content-Type: application/graphql+json` since any compliant server must support this type.
+
+This header MAY include encoding information (e.g. `Content-type: application/json; charset=utf-8`)
```

Comment:
Since the one required content-type is `graphql+json`, should the example of additionally supplying `charset` be based on that required content-type?  (Or was your intent to show both `other content type` and `encoding information` in a single example?)  If this is only the example for encoding information then:
 
```suggestion
This header MAY include encoding information (e.g. `Content-type: application/graphql+json; charset=utf-8`)
```

Reply:
```suggestion
This header MAY include encoding information (e.g. `Content-Type: application/json; charset=utf-8`)
```